lib/matplotlib/dates.py

```diff
@@ -506,7 +519,7 @@ def num2date(x, tz=None):
         Number of days (fraction part represents hours, minutes, seconds)
         since the epoch.  See `.get_epoch` for the
         epoch, which can be changed by :rc:`date.epoch` or `.set_epoch`.
-    tz : str, default: :rc:`timezone`
+    tz : str or `~datetime.tzinfo`, default: :rc:`timezone`
         Timezone of *x*.
 
```

Comment:
```suggestion
    tz : str or `~datetime.tzinfo`, default: :rc:`timezone`
        Timezone of *x*.  If a string, *tz* is passed to `dateutil.tz`.  

```

(assuming we can resolve "dateutil" in our docs).

Reply:
Gah! I left those...

Reply:
I think we can. I was surprised to see another class doing something like this, so I checked it out.